### ISSUE TITLE
Feature Request: Language Server

### ISSUE THREAD

**Issue by jansol** [evrsen/zed-sql#2]
I'd love to see language server support getting added to this! As far as I can tell the most promising LS so far is [sqls](https://github.com/sqls-server/sqls) and it seems to also support several DBMS.

I'm also partial to [postgres_lsp](https://github.com/supabase-community/postgres_lsp) but that still seems to be in a very early stage of development and is limited to Postgres which is probably not ideal for this extension.

(There is also a node-based SQL LS which I assume is what vscode uses, but that involves pulling down yet another version of the node runtime, which I'm not a huge fan of)

**Comment by evrsen** [comment]
Hi, thanks for reaching out! I'll take a look and see if I can get the lsp integration implemented this or next week, i've created a PR that you can follow and contribute to if you'd wish. 👌

- https://github.com/evrsen/zed-sql/pull/3

**Comment by h14h** [comment]
Bumping this issue.

Writing SQL migrations is much nicer with proper linting, since I do it infrequently enough that I find myself messing up the syntax more often than in other languages 😅 

It's not super urgent, but this is one of the last missing features of Zed that has me periodically hopping back over to neovim.

**Comment by ahmedmbarektorjmene** [comment]
yes even me trying to return to vscode because of sql language server is still not supported.
